xbmc/input/linux/LinuxInputDevices.cpp

```diff
@@ -460,6 +461,13 @@ bool CLinuxInputDevice::KeyEvent(const struct input_event& levt, XBMC_Event& dev
 {
   int code = levt.code;
 
+  if((levt.code == BTN_TOUCH || levt.code == BTN_TOOL_FINGER || levt.code == BTN_MOUSE)
+      && levt.value == 1 && (levt.type == EV_ABS || g_advancedSettings.m_TouchMouse))
+  {
+    m_mouseX = g_graphicsContext.GetWidth() + 2;
```

Comment:
Hi Pär Björklund!

Thank you for your comment on my first ever made push request.
The changes I've made originally come from
http://www.engineering-diy.blogspot.ro/ were we try to use a raspberry pi
for a Car PC that uses kodi to interact with all stuff a radio should
handle.

I will strip down everything that is not necessarry on my side (like the
block you mentioned above.)
Should I create a new pull request, or is an modification of the existing
preferred?

best regards
Mario Holzinger

2015-08-31 23:01 GMT+02:00 Pär Björklund <notifications@github.com>:

> In xbmc/input/linux/LinuxInputDevices.cpp
> <https://github.com/xbmc/xbmc/pull/7940#discussion_r38358896>:
>
> > @@ -460,6 +461,13 @@ bool CLinuxInputDevice::KeyEvent(const struct input_event& levt, XBMC_Event& dev
> >  {
> >    int code = levt.code;
> >
> > +  if((levt.code == BTN_TOUCH || levt.code == BTN_TOOL_FINGER || levt.code == BTN_MOUSE)
> > +      && levt.value == 1 && (levt.type == EV_ABS || g_advancedSettings.m_TouchMouse))
> > +  {
> > +    m_mouseX = g_graphicsContext.GetWidth() + 2;
>
> If this is accepted the +2 needs some comments
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/xbmc/xbmc/pull/7940/files#r38358896>.
>